### ISSUE TITLE
Once again, I have a DMXOPL update 

### ISSUE THREAD

**Issue by sneakernets** [Wohlstand/libADLMIDI#44]
I will be taking a break from DMXOPL a bit to work on other projects, so I think it's safe enough to add this git revision in ADLMIDI:

https://github.com/sneakernets/DMXOPL/blob/e624951fd9f5603fdb6f601132d8c4b4d4a09667/GENMIDI(GS).wopl

I wasn't able to add the new "blank" flags, so that might be an issue, but this is largely so GZDoom can have a version with my new Synth Bass 2 patch, which BTSX uses *a lot*.

Thanks!

**Comment by Wohlstand** [comment]
About of blank flags: you must take the latest bank editor assembly and call the "Edit" -> "Re-Calculate sounding delays" menuitem where blank flags are will be automatically set to all instruments are gave no sound. The core of libADLMIDI now supports them, and I also sent update to SDL Mixer X and to GZDoom too. I think, I'll need also to correct the gen_adldata's measurer where on editor side I have fixed the detection of a silence which was inaccurate.

**Comment by sneakernets** [comment]
Oh okay!  I recalculated the Bank. Here: https://github.com/sneakernets/DMXOPL/commit/3b66f53b22173e823dbbee6bfda15e32d7624704

**Comment by Wohlstand** [comment]
Also, is fallback now works fine at you? (with that MIDI file where French horn was silent) (with latest libADLMIDI and adlmidiplay)

**Comment by Wohlstand** [comment]
Just now I sent your update to everywhere: here, gzdoom, AudioCodecs

**Comment by Wohlstand** [comment]
Try to open a file I have put into my fm banks folder, is its a new version or something older? Ini file now refers GS version, but gen adldata ignores all extra banks and takes first two only as you already know.

**Comment by sneakernets** [comment]
I see two files in the wopl folder:
`DMXOPL3-by-sneakernets.wopl`
and
`DMXOPL3-by-sneakernets-GS.wopl`
The GS one has the guiros I made recently, the other file doesn't.  My file naming on my repository isn't the best, they still have GENMIDI names, which aren't even needed anymore. I'll do some spring cleaning and figure out what best names fit my files.

But the "GS" one is the newest, and I'm not sure if that one is being "compiled" or not. I'm not around a computer with a compiler to check.

**Comment by Wohlstand** [comment]
When multi-bank files are imported as embedded banks, there are stripping up to GM banks as that stuff doesn't allows multi-bank yet. Custom-loaded banks are supporting multi-banks well.

P.S. I suggesting you to call `make clean` to guarantee the build will use new re-generated bank. Also, re-call CMake with enabling of adldata re-generation.

**Comment by sneakernets** [comment]
sure, but I haven't made additional percussion banks yet, or at least I don't think I have. That's why I was confused.

**Comment by Wohlstand** [comment]
> I haven't made additional percussion banks yet, or at least I don't think I have

In parallel, I experimenting with multi-percussions at OPNMIDI where I made xg.wopn file with multiple percussion banks and one of is TR-808 which is best comparison to original sounding than other banks. Percussions are has different strategy of bank selection by using of patch number as bank number.

> Ah, it's working now. Then why are the GZDoom dev versions playing the older bank?
Oh dear.

As just now I have returned to my home, I gonna to check out the dev build of GZDoom for sounding, and also check out for reason of [one weird bug](https://forum.zdoom.org/viewtopic.php?f=104&t=59997&p=1047159) which people seems have figured out
